_docs_v5/installation.md

```diff
@@ -34,8 +52,65 @@ REQUESTS_CA_BUNDLE<br/>
 
 <p style="margin-bottom:20px;">
 Here are some directions on environment variables in Windows:
-<a href="https://www.computerhope.com/issues/ch000549.htm">Windows Environment Variables</a><br/>
+  <a href="https://www.computerhope.com/issues/ch000549.htm">
+    Windows Environment Variables
+  </a>
+<br/>
 You will need to create new ones with the names above and set their value to the full path to the certificate file.
 </p>
-<p style="margin-bottom:20px;">After these changes the installer should work. At Ball please contact <a href="mailto:cosmos@ballaerospace.com">cosmos@ballaerospace.com</a> for assistance.</p>
+
+<p style="margin-bottom:20px;">
+  At Ball please contact
+  <a href="mailto:cosmos@ballaerospace.com">
+    cosmos@ballaerospace.com
+  </a> for assistance.
+</p>
 </div>
+
+### RUN
+
+Run `cosmos-control.bat` start (Windows), or `cosmos-control.sh` start (linux/Mac)
+
+> If you see an error indicating docker daemon is not running ensure Docker and Docker compose is installed and running. If it errors please try to run `docker --version` or `docker-compose --version` and try to run the start command again. If the error continues please include the version in your issue if you choose to create one.
+
+> `cosmos-control.*` can help solve some of the short falls of docker-compose when building the containers.
+
+> `cosmos-control.*` takes multiple arguments. Run with no arguments for help. An example run of cosmos-control.bat with no arguments will show a usage guide. 
+```
+.\cosmos-control.bat
+Usage: "cosmos-control.bat" [start, stop, cleanup, build, run, deploy, util]
+*  build: build the containers for cosmos
+*  start: run the docker containers for cosmos
+*  stop: stop the running docker containers for cosmos
+*  restart: stop and run the minimal docker containers for cosmos
+*  cleanup: cleanup network and volumes for cosmos
+...
+```
+
```

Comment:
Please add a "note warning" that if you get a docker error when building cosmosc2-minio-init:latest it could be due to an issue downloading the minio container. Run "docker pull minio/mc:RELEASE.2021-11-05T10-05-06Z" and then re-run cosmos-control start